jupyter-js-widgets/package.json

```diff
@@ -54,8 +54,7 @@
     "webpack": "^1.12.11"
   },
   "dependencies": {
-    "backbone": "1.2.0",
-    "bootstrap": "^3.3.5",
+    "backbone": "^1.2.3",
```

Comment:
(From 1.2.0 to 1.2.2)
On Mar 17, 2016 8:29 AM, "Jonathan Frederic" <notifications@github.com>
wrote:

> In jupyter-js-widgets/package.json
> <https://github.com/ipython/ipywidgets/pull/400#discussion_r56495613>:
>
> > @@ -54,8 +54,7 @@
> >      "webpack": "^1.12.11"
> >    },
> >    "dependencies": {
> > -    "backbone": "1.2.0",
> > -    "bootstrap": "^3.3.5",
> > +    "backbone": "^1.2.3",
>
> We saw the same kind of breaking change from Backbone 1.1.0 to 1.2.0 IIRC
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ipython/ipywidgets/pull/400/files/93d072476e3f854d4a1bf870bd1b50d572a8643a#r56495613>
>